modules/transactions/src/integrationTest/java/org/apache/ignite/internal/disaster/ItDisasterRecoveryReconfigurationTest.java

```diff
@@ -1267,7 +1267,7 @@ void testAssignmentsChainUpdate() throws Exception {
 
         assertStableAssignments(node0, partId, initialAssignments);
 
-        assertAssignmentsChain(node0, partId, AssignmentsChain.of(initialAssignments));
+        assertAssignmentsChain(node0, partId, AssignmentsChain.of(-1, -1, initialAssignments));
```

Comment:
I didn't get why you need default term and index (there's another comment about that above). However, If you do really need them, why not to use 
`public static AssignmentsChain of(Assignments... assignments) {`
that you've introduced?